libs/shared-components/src/connected-accounts/connected-account-list-item.tsx

```diff
@@ -52,7 +91,7 @@ const CAListItem: FC<CAListProps> = ({
   const [isListOpened, setIsListOpened] = useState(false)
   const [burgerRects, setBurgerRects] = useState<DOMRectList | null>(null)
   const ref = useRef<HTMLDivElement>(null)
-  useOutside(ref, () => setIsListOpened(false), trackingRefs)
+  useOutside(ref, () => setTimeout(() => setIsListOpened(false), 500), trackingRefs)
```

Comment:
React's error "Can't perform a React state update on an unmounted component" is possible here without unsubscribing. Not critical.